### PR TITLE
V5.2/bugfix/pbld 146 srp shader alpha clip

### PR DESCRIPTION
**Pull request by modrimkus-unity**
### Purpose of this PR

Breaks SRP samples into 2019 specific and 2020+. ShaderGraphs authored in 2019.X do not migrate well to 2020.X+ as the process enables alpha clipping and bumps render queue to 2450.

### Links

**Jira:** https://jira.unity3d.com/browse/PBLD-147

### CODE REVIEW

File: Samples~/HDRP/.sample.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{
-	"displayName":"HDRP Support",
-	"description": "Shaders and materials for the High Definition Render Pipeline.",
-	"createSeparatePackage": false
-}
+{
+	"displayName":"HDRP Support",
```
Comment (lopezt-unity) [inline Samples~/HDRP/.sample.json:2]: Should this be "HDRP Support 2020 and later" ?

File: Samples~/UniversalRenderPipeline/.sample.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
-{
-	"displayName":"Universal Render Pipeline Support",
-	"description": "Shaders and materials for the Universal Render Pipeline.",
-	"createSeparatePackage": false
-}
+{
+	"displayName":"Universal Render Pipeline Support",
```
Comment (lopezt-unity) [inline Samples~/UniversalRenderPipeline/.sample.json:2]: Should this be "URP Support for 2020 and later" ?

**Review by lopezt-unity — APPROVED** [review]
LGTM, minor comments on the display names of these samples

File: Samples~/UniversalRenderPipeline/.sample.json (hunk 2 above)
Comment (modrimkus-unity) [inline Samples~/UniversalRenderPipeline/.sample.json:2]:
```suggestion
	"displayName":"URP Support for 2020.X and later",

```

File: Samples~/UniversalRenderPipeline 2019/.sample.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+	"displayName":"Universal Render Pipeline Support 2019.X",
```
Comment (modrimkus-unity) [inline Samples~/UniversalRenderPipeline 2019/.sample.json:2]:
```suggestion
	"displayName":"URP Support for 2019.X",
```

File: Samples~/HDRP/.sample.json (hunk 1 above)
Comment (modrimkus-unity) [inline Samples~/HDRP/.sample.json:2]:
```suggestion
	"displayName":"HDRP Support for 2020.X and later",

```

**Pull request merged**